api/views.py

```diff
@@ -1,26 +1,19 @@
-from django.contrib.auth.models import CustomUser, Zone, ZipZone, Plant, Slot, PlantZone, PlantSlot
+from .models import Zone, Plant, Slot, PlantZone, PlantSlot
 from rest_framework import generics
-from api.serializers import PlantSerializer, SlotSerializer, ZoneSerializer, CustomUserSerializer, PlantZoneSerializer, PlantSlotSerializer, PlantSlotActivity
+from rest_framework.response import Response
+from api import serializers
 from django.shortcuts import get_list_or_404, get_object_or_404
+from django.contrib.auth import get_user_model
 
+User = get_user_model()
 
 class ListAvailablePlants(generics.ListAPIView):
-    queryset = Plant.objects.all()
-    serializer_class = PlantSerializer
+    queryset = PlantZone.objects.all()
+    serializer_class = serializers.SuggestedPlantSerializer
 
     def get(self, request):
-        #user = CustomUser.objects.get(id = user_id)
-        user = get_object_or_404(CustomUser, id=self.request.user.pk)
-        available_plants = user.zone.all_plants_in_a_zone()
-        serializer = PlantZoneSerializer(available_plants)
+        user = User.objects.get(pk=1)
+        plant_zones = user.zone.plants.exclude(calendar__contains=','*11)
```

Comment:
will this line work? currently a user has no relationship to zones

Reply:
So I might have no been clear enough here.

`python manage.py createsuperuser` defaults to Null -- to make that command work there was too much overriding against the Custom User model. But POSTing to `/api/user/create/` will attach the zone to a user. But for right now, when you create a super user you have to manually login in to the admin dashboard and set it yourself before performing that GET against `/api/suggested/`.

Reply:
got it.